docs/index.asciidoc

```diff
@@ -449,38 +449,39 @@ here, ":target_id" is a named parameter. You can configure named parameters
 with the `parameters` setting.
 
 [id="plugins-{type}s-{plugin}-statement_filepath"]
-===== `statement_filepath` 
+===== `statement_filepath`
 
   * Value type is <<path,path>>
   * There is no default value for this setting.
 
 Path of file containing statement to execute
 
 [id="plugins-{type}s-{plugin}-tracking_column"]
-===== `tracking_column` 
+===== `tracking_column`
 
   * Value type is <<string,string>>
   * There is no default value for this setting.
 
-If tracking column value rather than timestamp, the column whose value is to be tracked
+The column whose value is to be tracked (if tracking column value rather than the last time the query was run)
```

Comment:
How about "The column whose value is to be tracked if use_column_value is set to true" ?